packages/ic-management/src/ic-management.canister.ts

```diff
@@ -79,7 +79,9 @@ export class ICManagementCanister {
     settings,
     senderCanisterVersion,
   }: CreateCanisterParams = {}): Promise<Principal> => {
-    const { canister_id } = await this.service.create_canister({
+    const { create_canister } = this.service;
```

Comment:
I'm not really a fan of this syntax. Is it not possible to fix your IDE configuration?

Reply:
It's the syntax we already use, here I'm updating legacy code. It's a question of opinion and I rather like this syntax.